Introduction

PatrickCQ opened the pull request:
Update of the language what this document is about.

PatrickCQ committed:
update abstract

Easier to read.
Made clear many transport protocols may be described.
Made clear that ontologies are described separately.

PatrickCQ committed:
update introduction

summary of plan from https://github.com/w3c/automotive/blob/gh-pages/charter-2018.html

LaurentCQ commented on spec/Gen2_Core.html:
```diff
@@ -39,14 +39,34 @@
-        This document, the Gen2 core specification, describes the Gen2 messaging layer. The companion specification Gen2 transport details the mapping of the messaging layer to selected transports.
+        This document, the Gen2 core specification, describes the Gen2 messaging layer. 
+        The companion specifications for the Gen2 transport protocols detail the mapping of the messaging layer to selected transports.
+        The ontologies based on the Gen2 core specifation are defined in documents for the specific domain. 
```
Typo: specifation -> specification.

PatrickCQ committed:
update intro typo and language and add examples for domains

PatrickCQ committed:
update describe roles for messages and data

It must be clear what part is just transport and what part is data description/structuring data.

PatrickCQ committed:
update client using data

it is not defined how a client has to use the data (manipulating describes changing).
